src/website/views/components/forms/number-input.pug

```diff
@@ -56,119 +104,189 @@ p.-text
         <input type="number" class="a-input -md" value="0" aria-label="Input Label">
-        <button aria-label="Decrease"></button>
-        <button aria-label="Increase"></button>
+        <button class="a-btn -icon" aria-label="Decrease">
+          <div class="a-btn__content">
+            <i class="a-icon">
+              <svg aria-hidden="true">
+                <use xlink:href='#icon-triangle-down'/>
+              </svg>
+            </i>
+          </div>
+        </button>
+        <button class="a-btn -icon" aria-label="Increase">
+          <div class="a-btn__content">
+            <i class="a-icon">
+              <svg aria-hidden="true">
+                <use xlink:href='#icon-triangle-up'/>
+              </svg>
+            </i>
+          </div>
+        </button>
       </div>
     </div>
-    <div class="m-form__item -row -inline">
-      <div class="m-input__wrapper">
+    <div class="m-form__item -inline">
+      <div class="m-input-number">
         <input type="number" class="a-input -lg" value="0" aria-label="Input Label">
-        <button aria-label="Decrease"></button>
-        <button aria-label="Increase"></button>
+        <button class="a-btn -icon" aria-label="Decrease">
+          <div class="a-btn__content">
+            <i class="a-icon">
+              <svg aria-hidden="true">
+                <use xlink:href='#icon-triangle-down'/>
+              </svg>
+            </i>
+          </div>
+        </button>
+        <button class="a-btn -icon" aria-label="Increase">
+          <div class="a-btn__content">
+            <i class="a-icon">
+              <svg aria-hidden="true">
+                <use xlink:href='#icon-triangle-up'/>
+              </svg>
+            </i>
+          </div>
+        </button>
       </div>
     </div>
-    <div class="m-form__item -row -inline">
-      <div class="m-input__wrapper">
+    <div class="m-form__item -inline">
+      <div class="m-input-number">
         <input type="number" class="a-input -xl" value="0" aria-label="Input Label">
-        <button aria-label="Decrease"></button>
-        <button aria-label="Increase"></button>
+        <button class="a-btn -icon" aria-label="Decrease">
+          <div class="a-btn__content">
+            <i class="a-icon">
+              <svg aria-hidden="true">
+                <use xlink:href='#icon-triangle-down'/>
+              </svg>
+            </i>
+          </div>
+        </button>
+        <button class="a-btn -icon" aria-label="Increase">
+          <div class="a-btn__content">
+            <i class="a-icon">
+              <svg aria-hidden="true">
+                <use xlink:href='#icon-triangle-up'/>
+              </svg>
+            </i>
+          </div>
+        </button>
       </div>
     </div>
 
 h2 Expanded version
 p.-text
-  | There is available an expanded version of the number input component.
+  | Apply the <code>-alt</code> class to render alternative number input version.
```

Comment:
The custom element is still using the attribute `expanded`. Can we unify the two? We'll need something more descriptive than `alt` incase we introduce another variant.

Reply:
@mattnickles makes sense. What about -inline? Any other suggestion?